### DISCUSSION TITLE
neThing.xyz - Text-to-3D Generative AI

### DISCUSSION THREAD

**Discussion by rawwerks** [google/model-viewer#4652] [Show and tell]
I'm using model-viewer to power https://neThing.xyz/ (pronounced "any thing dot x,y,z"), which is a coding-based approach to text-to-3d generative AI. 

<img width="2019" alt="image" src="https://github.com/google/model-viewer/assets/19483938/9d9b6191-df1c-4845-bdaa-f0ea81bdacab">

The app is currently free to use with a Discord or GitHub login.

This is a brand new project and to be honest the AI is not very "smart" yet. I would greatly appreciate any feedback you might have, either here or via our feedback form: https://forms.gle/sVbapBRsuEBNJb176

**Comment by elalish** [comment]
Nice - if you use Twitter, feel free to @modelviewer and we'll repost.

**Reply by rawwerks** [reply]
@elalish - sweet, thank you so much for the offer. i have a few final touches to implement before i'm ready for the spotlight, but i will definitely tag `@modelviewer` when i post.
